### PR TITLE
adding_new_levels_2

### PR DESCRIPTION
**Pull request by shoc71**
- 2 new levels added to the half-screen
- 1 new level added to block maze

### CODE REVIEW

File: Keep On Going/assets/levels/008_Block-Maze.txt [hunk 1]
```diff
@@ -419,4 +419,126 @@ self.win_zones =  [
         pygame.draw.rect(screen, (47, 237, 237), [1056, 443, 9, 28])
 ]
 
-=====
\ No newline at end of file
+==========
+
+Block_Maze_4 (851, 507, 1)
```
Comment (slysnivy) [inline Keep On Going/assets/levels/008_Block-Maze.txt:424]: There's a space in between the Block_Maze_4 and (851, 507, 1). I don't know if the file loader will recognize this. Therefore it should be fixed in a mini commit.

**Review by slysnivy — COMMENTED** [review]
There is one minor issue that may or may not cause issues when loading Block Maze 4. Regardless, I suggest fixing this in a mini commit (in the same PR)

File: Keep On Going/assets/levels/008_Block-Maze.txt (hunk 1 above)
Comment (shoc71) [reply Keep On Going/assets/levels/008_Block-Maze.txt]: thanks for the catch

**Review by slysnivy — COMMENTED** [review]
Ok, looks great.

**Pull request merged**